client/my-sites/plugins/plugin-action/style.scss

```diff
@@ -64,7 +65,6 @@
 .plugin-action__disabled-info.info-popover {
 	float: right;
 	height: 16px;
-	width: 24px;
```

Comment:
Why does this need to be removed? It seems to cause a misalignment on the manage page when the viewport is less than 1040px.

<img width="690" alt="screen shot 2017-09-20 at 14 26 55" src="https://user-images.githubusercontent.com/908665/30646203-21e67ba8-9e10-11e7-95a9-0d80031f9092.png">

Reply:
Without it the icons aren't aligned on the plugin detail page. :(

I'm not sure what the best alternative is here.

Reply:
@jeherve I'd suggest keeping the width setting as it is, and instead, change the `translateX` value to `0` like this `transform: translate(0, -2px);` for `.plugin-action__disabled-info.info-popover .gridicons-info-outline` (Just a few lines below)

Reply:
It seems to be hard to find something that will work everywhere.

With 
```css
.plugin-action__disabled-info.info-popover .gridicons-info-outline {
	transform: translate(2px, -2px);
}
```
- Plugin detail page looks good:
![screenshot 2017-09-21 at 15 51 15](https://user-images.githubusercontent.com/426388/30699379-ea7258ba-9ee4-11e7-958f-1264ddaf3e19.png)
- Plugins list is not perfectly aligned:
![screenshot 2017-09-21 at 15 51 39](https://user-images.githubusercontent.com/426388/30699427-0534e186-9ee5-11e7-9d1c-be71c84e601d.png)

With
```css
.plugin-action__disabled-info.info-popover .gridicons-info-outline {
	transform: translate(0, -2px);
}
```

- Plugin detail page has a small issue:
![screenshot 2017-09-21 at 15 47 05](https://user-images.githubusercontent.com/426388/30699478-2562da30-9ee5-11e7-9dd6-fe02ec47a6fb.png)
- Plugins list looks good:
![screenshot 2017-09-21 at 15 46 18](https://user-images.githubusercontent.com/426388/30699489-2d21795c-9ee5-11e7-8120-b91381ac9b56.png)